.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Issue Report
+description: Report an unexpected behavior or failure (AKA a bug)
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+          * Please read these [Tips](https://docs.elementary.io/contributor-guide/feedback/reporting-issues)
+          * Be sure to search open and closed issues for duplicates
+
+  - type: textarea
+    attributes:
+      label: What Happened?
+      description: A detailed description of unexpected behavior will help us address your issue more quickly. Do your best!
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Explain the exact steps one would take to experience the issue. If applicable, add screenshots or screen recordings.
+      placeholder: |
+          1. Went here 
+          2. Did this 
+          3. Clicked that 
```

Comment:
Ah, since they're using an actual placeholder attribute, we can't get multi-line here. It's probably fine (and correctly collapses down to one line in the form), but something to note.